sos/policies/runtimes/crio.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2021 Red Hat, Inc., Nadia Pinaeva <npinaeva@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.runtimes import ContainerRuntime
+from sos.utilities import sos_get_command_output
+from pipes import quote
+
+
+class CrioContainerRuntime(ContainerRuntime):
+    """Runtime class to use for systems running crio"""
+
+    name = 'crio'
+    binary = 'crictl'
+
+    def get_containers(self, get_all=False):
+        """Get a list of containers present on the system.
+
+        :param get_all: If set, include stopped containers as well
+        :type get_all: ``bool``
+        """
+        containers = []
+        _cmd = "%s ps %s" % (self.binary, '-a' if get_all else '')
+        if self.active:
+            out = sos_get_command_output(_cmd, chroot=self.policy.sysroot)
+            if out['status'] == 0:
+                for ent in out['output'].splitlines()[1:]:
+                    ent = ent.split()
+                    # takes the form (container_id, container_name)
+                    containers.append((ent[0], ent[-3]))
```

Comment:
As the `-3` is the only difference from the parent original method: does not it make sense to have the index as a class attribute like `binary`, defaulting to `-1` and overwritten in `CrioContainerRuntime` to `-3`?

Reply:
Agreed. I think having these indexes as a part of the class would simplify things.

Reply:
I'm not sure about how many other runtimes we'll have so that this option is used more than once and how general this suggestion that all runtimes `ps` output is only different in columns order (because that's the only when this option will work), but if you think it's reasonable, let's do that.

Reply:
Fair point. At the moment while it is duplication, it is small and confined and I don't see future expansion without further adding new runtimes, at which point we can re-evaluate this override or the suggestion above.

Reply:
ACK